docs/pages/enterprise/sso/google-workspace.mdx

```diff
@@ -27,95 +27,107 @@ to define policies like:
 
 Before you get started you’ll need:
 
-- An Enterprise version of Teleport downloaded from [https://dashboard.gravitational.com/](https://dashboard.gravitational.com/web/login).
-- Be a Google Workspace Super Admin. As Google Best Practices, we would recommend setting up a separate super admin with 2FA vs using your user.
-  - e.g. A dedicated account [ben-ops@practice.io](mailto:ben-ops@practice.io) vs my daily [ben@practice.io](mailto:ben@practice.io)
-- Ability to create GCP Project.
-  - This might require signing up to GCP, but for this project it won’t require using any paid services. It’s just a side effect of G Suite and GCP being closely related.
-- Have a [verified Domain](https://support.google.com/a/answer/60216?hl=en).
-- Ability to Setup Google Workspace Groups
-
-## Configure G Suite
-
-1. Obtain OAuth 2.0 credentials
-   [https://developers.google.com/identity/protocols/OpenIDConnect](https://developers.google.com/identity/protocols/OpenIDConnect)
-2. Create a new Project.
-   ![Create New Project](../../../img/gsuite/gsuite-1-new-project.png)
-3. Select OAuth client ID.
-   ![Create OAuth Creds](../../../img/gsuite/gsuite-2-created-creds.png)
-4. Make Application Type Public & Setup Domain Verification
-   ![Setup Application Type](../../../img/gsuite/gsuite-3-oauth.png)
-5. Copy OAuth Client ID and Client Secret for YAML Below.
-   Note: The redirect_url: `https://teleport.example.com:3080/v1/webapi/oidc/callback`
-
-![Copy Client Secret](../../../img/gsuite/gsuite-5-copy-client-id.png)
-
-## Create a Service Account
-
-![Create OAuth Creds](../../../img/gsuite/gsuite-5a-service-account.png)
-
-Leave Service account users roles, and admin roles as blank.
-
-![Create OAuth Creds](../../../img/gsuite/gsuite-5b-service-account.png)
-
-Leave Service account permissions as blank.
-
-![Create OAuth Creds](../../../img/gsuite/gsuite-5c-service-account.png)
-
-### Enable Account Delegation
-
-![Create OAuth Creds](../../../img/gsuite/gsuite-5d-service-account-delegation.png)
-
-![Create OAuth Creds](../../../img/gsuite/gsuite-5e-enable-delegation@2x.png)
-
-### Download Service Account JSON
-
-![Create OAuth Creds](../../../img/gsuite/gsuite-5f-download-json.png)
-
-This JSON file will need to be uploaded to the Authentication server, and will be later referenced by
-the OIDC Connector, under `google_service_account_uri` or inline with `google_service_account`.
+- An Enterprise version of Teleport downloaded from [https://dashboard.gravitational.com/](https://dashboard.gravitational.com/).
+- A Google Workspace Super administrator account. We recommend setting up a separate super admin account with 2FA as opposed to granting your daily user super admin privileges.
+   Use a dedicated account, e.g., `ben-ops@example.com`, as opposed to your daily `ben@example.com` account.
```

Comment:
```suggestion

Use a dedicated account, e.g., `ben-ops@example.com`, as opposed to your daily `ben@example.com` account.
```
I think this bullet is a bit long, so I've suggested adding another line.

Reply:
Honestly, "we recommend setting up a separate super admin account" should be enough; using something like `ben-ops` is probably a bad idea compared to `teleport-auth-helper` anyway, so I just removed the example.